modules/email/src/admin/index.ts

```diff
@@ -83,6 +84,7 @@ export class AdminHandlers {
         {
           path: '/templates',
           action: ConduitRouteActions.POST,
+          description: `Creates a new email template requiring at least template's name and body.`,
```

Comment:
Shouldn't elaborate on param requirements.
This can change independently of the description and are already marked as required or optional in the route doc.